config/tensorleap.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: helm.cattle.io/v1
```

Comment:
Are you sure that you wanted to add this file to this PR?

Reply:
Yes, otherwise I'll have to to inline this in the instructions